sklearn/utils/_show_versions.py

```diff
@@ -71,15 +73,20 @@ def get_version(module):
 
 
 def show_versions():
-    "Print useful debugging information"
+    """Print useful debugging information"""
 
     sys_info = _get_sys_info()
     deps_info = _get_deps_info()
 
-    print('\nSystem:')
+    print('\nSystem')
+    print('------')
     for k, stat in sys_info.items():
         print("{k:>10}: {stat}".format(k=k, stat=stat))
 
-    print('\nPython deps:')
+    print('\nPython deps')
+    print('-----------')
     for k, stat in deps_info.items():
         print("{k:>10}: {stat}".format(k=k, stat=stat))
+
+    print("\n{k:>10}: {stat}".format(k="Built with OpenMP",
+                                     stat=_openmp_supported()))
```

Comment:
as discussed with @ogrisel irl, I added an entry in show_versions to show OpenMP status.
It will displayed as `Built with OpenMP: True/False`.
Later it could be improved to display the OpenMP runtime linked (requires threadpoolctl)

I also allowed myself a little cosmetic change in this function. replaced
```
Python deps:
       pip: 18.1
setuptools: 41.4.0
   sklearn: 0.22.dev0
       ...
```
by
```
Python deps
-----------
       pip: 18.1
setuptools: 41.4.0
   sklearn: 0.22.dev0
       ...
```
which I find nicer.

Reply:
Python deps => Python dependencies